entrypoint.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Collecting static files..."
+uv run python manage.py collectstatic --noinput
+
+echo "Starting Gunicorn server..."
+uv run gunicorn project.wsgi:application \
+    --bind 0.0.0.0:${PORT:-8000} \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)